### PR TITLE
chore: Remove old like functionality from MessageToolBoxView - WPB-2958

### PR DESCRIPTION
**Pull request by agisilaos**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-2958" title="WPB-2958" target="_blank">TaskWPB-2958</a>  [iOS] Remove Old Like functionality from MessageToolBoxView
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we get rid of the old "like functionality" from the MessageToolBoxView since it's not needed anymore.  

--

A PR will follow up moving the Date and status of the message to a different view. 

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Review by netbe — APPROVED** [review]
- looks like font change on some screenshots

**Comment by agisilaos** [comment]
> - looks like font change on some screenshots
> 
> 

That's correct. When initially made the changes (during accessibility sprints) we didn't update the tests. 

Aka the tests didn't fail. I think that's kind of a regression from FB's side because I observed the same behavior in another case as well. 

You have to manually add `recordMode = true` and run the tests again. @netbe

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift [hunk 1]
```diff
@@ -281,9 +278,8 @@ final class MessageToolboxView: UIView {
                 self.statusSeparatorLabel.isHidden = true
                 self.countdownLabel.isHidden = true
             }
-        case .reactions(let reactionsString):
+        case .reactions:
```
Comment (johnxnguyen) [inline wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift]: Perhaps we should remove this as a datasource.

**Review by johnxnguyen — CHANGES_REQUESTED** [review]

File: wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift (hunk 1 above)
Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift]: I was thinking about the same thing. Let me check if it impacts any other view and I'm planning to remove it. @johnxnguyen.

Comment (agisilaos) [reply wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/Utility/MessageToolboxView.swift]: @johnxnguyen Made all the appropriate changes. Ready for another round of review!

**Review by johnxnguyen — APPROVED** [review]

**Pull request merged**